x-pack/platform/plugins/private/reindex_service/server/src/lib/reindex_service.test.ts

```diff
@@ -100,41 +99,13 @@ describe('reindexService', () => {
         { has_all_requested: true }
       );
 
-      const hasRequired = await service.hasRequiredPrivileges('anIndex');
```

Comment:
This service no longer performs a check on the calculated new index name - Upgrade Assistant calculates the new index name and must checks privs on it.